libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -558,7 +638,8 @@ impl<'a> JsiiImporter<'a> {
 			if let Some(params) = &initializer.parameters {
 				for (i, param) in params.iter().enumerate() {
 					// If this is a resource then skip scope and id arguments
-					if is_resource {
+					// TODO hack - skip this check if the resource's name is "App"
```

Comment:
Why BTW? What's wrong with a resource named `App`?

If this is for the SDK `App` resource, then we could just add `@hidden` attribute to it or something, no?

Reply:
https://github.com/winglang/wing/issues/1485

(I'll add a link to the issue in the comment)

Reply:
This is about importing other `App` constructs, like `cdktf.App` or `cdk8s.App`